Polyhedron/demo/Polyhedron/Plugins/PMP/Join_and_split_polyhedra_plugin.cpp

```diff
@@ -137,16 +139,23 @@ void Polyhedron_demo_join_and_split_polyhedra_plugin::on_actionSplitPolyhedra_tr
       {
         delete new_polyhedra.front();
         msg_interface->information( tr("%1 has only one connected component").arg(item->name()) );
+        QApplication::restoreOverrideCursor();
         continue;
       }
 
       int cc=0;
+      compute_color_map(item->color(), item->isItemMulticolor() ? new_polyhedra.size() : 1,
```

Comment:
`compute_color_map` takes an `unsigned int` as second argument, and `new_polyhedra.size()` is a `size_t`. There is a warning on Windows:
```
[100%] Building CXX object Plugins/PMP/CMakeFiles/join_and_split_polyhedra_plugin.dir/Join_and_split_polyhedra_plugin.cpp.obj
	cd [...]\test\Polyhedron_Demo\Plugins\PMP
	C:\PROGRA~2\MICROS~3.0\VC\bin\amd64\cl.exe  @C:\Users\Jane\AppData\Local\Temp\nm76D1.tmp
Join_and_split_polyhedra_plugin.cpp
E:\[...]\test\Polyhedron_Demo\Plugins\PMP\Join_and_split_polyhedra_plugin.cpp(148): warning C4267: 'argument': conversion from 'size_t' to 'unsigned int', possible loss of data
```

That is a different warning from the one fixed by f05bf5ba4f7d5c762a34042719abb2354ee527b5.